src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const object = {
+    result: 0,
+
+    operate(callback, number) {
+      callback(this, number);
```

Comment:
```suggestion
      callback.call(this, number);
```

Reply:
Yeap, it works. Thank you. But I don't really understand how it works. What does this '.call' mean? And, btw, I can remove 'return this' in methods and it still works.

Reply:
You will learn `call` method later. `return this` is needed to call methods in a chain